src/util/infoUtils.ts

```diff
@@ -20,16 +20,14 @@ export interface DisplayInfoCard {
 interface InfoFilterProps {
   accountFunded?: boolean
   buildNumber: string
+  cardInfos: InfoCard[]
```

Comment:
Before your changes in the previous PR, this was called `cards`. I would either go back to that name, or would use the name `infoCards` to match the type.

Reply:
We use this type for display and business logic so I wanted to kind of distance it from the cemented `InfoCard` type.

I'll revert to `cards`